windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
```

Comment:
Apparently the dependency library packages do not match those of Ubuntu (the Windshaft docs seem to assume Ubuntu.) How much do we care about making this work on Debian?